### DISCUSSION TITLE
Integration with Agregore

### DISCUSSION THREAD

**Discussion by RangerMauve** [freenet/freenet-core#501] [General]
Hey folks, I'm the maintainer of @AgregoreWeb and have been working on integrating various peer to peer protocols natively with web browsers via custom protocol handlers.

Would you be interested in chatting about speccing out a `locutus://` URL scheme as well as protocol handlers in Agregore so we can load apps and send data to contracts natively?

I think there's a lot of potential in mixing locutus with some other protocols like IPFS and having tools for deploying and interacting with Locutus apps right from within Agregore.

It'd be also interesting to talk about integrating it with Agregore Mobile down the line and taking advantage of the ad-hoc mesh network creation functionality we've been working on in there.

**Comment by RangerMauve** [comment]
Also it'd be interesting to talk about how we could use IPLD Databases in order to handle large volumes of data and have search indexes. https://blog.mauve.moe/posts/peer-to-peer-databases

**Comment by iduartgomez** [comment]
Not sure we are ready at this point to talk about integrations yet.

@sanity ?

Maybe should move this to ideas.

**Comment by RangerMauve** [comment]
Kinda along these lines, though. Is there a spec yet for linking to data stored in Locutus using URLs? That'd be the easiest path for me to start integrating it into the tools I've been building. :)
